docs/modules/getting_started/examples/code/getting_started.sh

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+if [ $# -eq 0 ]
+then
+  echo "Installation method argument ('helm' or 'stackablectl') required."
+  exit 1
+fi
+
+case "$1" in
+"helm")
+echo "Adding 'stackable-dev' Helm Chart repository"
+# tag::helm-add-repo[]
+helm repo add stackable-dev https://repo.stackable.tech/repository/helm-dev/
+# end::helm-add-repo[]
+echo "Installing Operators with Helm"
+# tag::helm-install-operators[]
+helm install --wait commons-operator stackable-dev/commons-operator --version 0.3.0-nightly
+helm install --wait secret-operator stackable-dev/secret-operator --version 0.6.0-nightly
+helm install --wait superset-operator stackable-dev/superset-operator --version 0.6.0-nightly
+# end::helm-install-operators[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
```

Comment:
It might be worth mentioning here (although we've made it clear in index.adoc that a cluster is assumed) that a Kubernetes cluster can be created by adding the `-k `option when using `stackablectl`

Reply:
okay, I think that could be added to the TIP. then it should also be mentioned that that requires kind and docker installed.

Reply:
Agreed.